### DISCUSSION TITLE
EEPROM placement on PicoCNC board

### DISCUSSION THREAD

**Discussion by JJTechman68** [phil-barrett/PicoCNC#20] [Q&A] [answered]
Does anybody have a picture for the orientation of the 8 pin EEPROM-FRAM chip for soldering onto the PicoCNC board?

**Comment by phil-barrett** [comment, marked as answer]
If you look closely, at U18, you will see a small circle in the upper left corner.  That is pin one.  You will also see a small indent on the EEPROM chip package at pin one.

![image](https://github.com/phil-barrett/PicoCNC/assets/16570958/30464c81-da5e-4a6d-ab6c-fc3bfd748f10)

**Reply by JJTechman68** [reply]
Thanks Phil!
